.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: '@wizbii/angular-pipes'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2.1.5
+        with:
+          node-version: '14'
+
+      - name: Cache Node Modules
+        id: cache-node-modules
+        uses: actions/cache@v2
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }}
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Lint lib
```

Comment:
What do you think about doing a separate job for lint test and build ?

Reply:
I don't know how to do it, make a PR on this after i merged ;)